manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -112,6 +112,7 @@ data:
       "dbdriver": "jdbc:postgresql://",
       "plots": "false",
       "local": "false",
+      "logAllHttpReqAndResp": "true",
```

Comment:
Why is this flag needed?

Reply:
Enabling debug is giving us a lot of logs which makes debugging tough. We're adding this to get the request and response data logged as INFO in logs when we need to debug any issue.
@msvinaykumar please review and confirm

Reply:
Yes, it's necessary. Currently, the debug flag contains an excessive amount of information, making it challenging to trace logs. However, the INFO level provides a well-balanced log. We now have INFO level for all requests and responses as well. Nonetheless, we can exclusively deactivate this feature using the environment variable logAllHttpReqAndResp.